tools/code_format/check_format.py

```diff
@@ -696,6 +747,17 @@ def checkSourceLine(line, file_path, reportError):
         reportError("Don't call grpc_init() or grpc_shutdown() directly, instantiate " +
                     "Grpc::GoogleGrpcContext. See #8282")
 
+  if not re.search("using .* = .*;", line):
+    smart_ptr_m = re.finditer("std::(unique_ptr|shared_ptr)<(.*?)>", line)
```

Comment:
what does the `_m` mean? Spell that out?

Reply:
done